src/k8s-engine/lima.ts

```diff
@@ -817,7 +826,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     });
   }
 
-  async start(config: { version: string; memoryInGB: number; numberCPUs: number; port: number; }): Promise<void> {
+  async start(config: Settings['kubernetes']): Promise<void> {
```

Comment:
Just realized that we're still unconditionally starting docker; we should only do that if we need it…

Reply:
Caught this.